web/modules/custom/asset_distribution/asset_distribution.module

```diff
@@ -70,3 +70,20 @@ function asset_distribution_rdf_load_bundle_alter($entity_values, &$bundle) {
     $bundle = \Drupal::entityTypeManager()->getStorage('rdf_type')->load('asset_release');
   }
 }
+
+/**
+ * Returns the solution that a release belongs to.
+ *
+ * @param \Drupal\rdf_entity\RdfInterface $asset_release
+ *    The asset release rdf entity.
+ *
+ * @return \Drupal\rdf_entity\RdfInterface
+ *    The solution rdf entity that the release is version of.
+ */
+function asset_distribution_get_release_solution(RdfInterface $asset_release) {
+  if ($asset_release->bundle() != 'asset_release') {
+    return NULL;
+  }
+  $target_id = $asset_release->field_isr_is_version_of->first()->target_id;
+  return Rdf::load($target_id);
```

Comment:
This is no longer needed, it has moved to `AssetDistributionRelations`. If you want to keep it so it can be used from procedural code, then you can replace the logic with a call to the service method, like is commonly done for example in `Drupal`.